assets/js/components/Popup.js

```diff
@@ -319,7 +324,22 @@ module.exports = class Popup {
     if (this.isOpen) {
       this.positionPopupHitBox(e);
     } else {
+      this.popupHitBox.setAttribute('aria-expanded', 'false');
+      this.popupHitBox.removeAttribute('tabindex', -1);
+      this.popupHitBox.setAttribute('aria-hidden', 'true');
       this.popupHitBox.style.display = 'none';
+      this.$link.focus();
+    }
+
+    if (this.isOpen) {
```

Comment:
This should be combined with the identical conditional on [line 324](https://github.com/elifesciences/pattern-library/pull/930/files#diff-752da1b00c0db6cd94aafc27aa81c946R324).

Reply:
The reason why I moved this outside was it was causing the triple click

Reply:
Resolved as did not work on Tuesday hence reason why I separated but now does since method created. Beginning to think my Mac is playing up on me.